mix.exs

```diff
@@ -21,20 +20,18 @@ defmodule NewRelixir.Mixfile do
   end
 
   defp deps do
-    [{:phoenix, "~> 1.3-rc"},
-     {:ecto, ">= 1.1.0 and < 3.0.0"},
-     {:newrelic, "~> 0.1.0"},
-     {:earmark, "~> 0.1", only: :dev},
-     {:ex_doc, "~> 0.11", only: :dev}]
+    [
+      {:earmark, "~> 0.1", only: :dev},
```

Comment:
The `earmark` library is already a dependency of `ex_doc`, we can skip it here.

Reply:
Done in 67751f1a3829e7b79319d39be03437c747b90ed2